specifyweb/frontend/js_src/lib/components/QueryBuilder/Export.tsx

```diff
@@ -87,18 +127,27 @@ export function QueryExportButtons({
         <QueryButton
           disabled={fields.length === 0}
           showConfirmation={showConfirmation}
-          onClick={(): void =>
-            doQueryExport(
-              '/stored_query/exportcsv/',
-              userPreferences.get(
-                'queryBuilder',
-                'behavior',
-                'exportFileDelimiter'
-              )
-            )
-          }
+          onClick={(): void => {
+            selectedRows.size === 0
+              ? doQueryExport(
+                  '/stored_query/exportcsv/',
+                  userPreferences.get(
+                    'queryBuilder',
+                    'behavior',
+                    'exportFileDelimiter'
+                  )
+                )
+              : downloadFile(
+                  `${
+                    queryResource.isNew()
+                      ? 'new query export'
+                      : queryResource.get('name')
+                  } - ${new Date().toDateString()}.tsv`,
+                  handleSelectedResults()
+                );
+          }}
         >
-          {queryText.createCsv()}
+          {commonText.export()}
```

Comment:
not sure how I feel about this change. @grantfitzsimmons opinions?

Reply:
Create CSV is much clearer. I think we should keep that.